tests/test_price_info.py

```diff
@@ -101,3 +101,17 @@ def test_price_info_str(price_info_trading):
     expected = "PythPriceInfo status PythPriceStatus.TRADING price 596.09162"
     assert str(price_info_trading) == expected
     assert repr(price_info_trading) == expected
+
+def test_price_info_to_json(price_info_trading):
```

Comment:
Would you be able to add a test which checks that the result of `to_json` is what we expect, instead of making assertions about the keys?

Reply:
I don't remember very well at this point, but I think I made it that way so that tests fail if a developer updates the class attributes without updating the to_json.